crates/database/src/byte_array.rs

```diff
@@ -67,10 +67,44 @@ impl<const N: usize> Encode<'_, Postgres> for ByteArray<N> {
     }
 }
 
+impl<'de, const N: usize> serde::Deserialize<'de> for ByteArray<N> {
```

Comment:
why to implement `serde` and not directly `FromRow`, so we don't need to parse the string directly but sqlx does the job?

Reply:
https://github.com/cowprotocol/services/pull/2809#discussion_r1675527726